src/admin/src/utils/AppUtil.js

```diff
@@ -28,3 +28,16 @@ export function dateCompare(a, b) {
   }
   return 0;
 }
+
+export function camelCaseToSentenceCase(input) {
```

Comment:
I think this might be way more complicated than it needs to be.

Couldn't you just do something simple like this?
```
function sentenceCase(input) {
    if (!input) {
        return "";
    }
    return input[0].toUpperCase() + input.substring(1).toLowerCase();
}
```